src/main/app/index.js

```diff
@@ -42,6 +42,39 @@ class App {
       app.commandLine.appendSwitch('enable-experimental-web-platform-features', 'true')
     }
 
+    app.on('second-instance', (event, argv, workingDirectory) => {
```

Comment:
Under what circumstances does it trigger the `second-instance` event?

Reply:
If you start a second instance on Linux and Windows (e.g. launch Mark Text again or click on a file in file manager/explorer, ...). I think macOS is handled by `open-file` by default.